core/async/src/multithread/runtime_handle.rs

```diff
@@ -12,11 +12,18 @@ pub(super) struct MultithreadRuntimeMessage<A> {
 /// for the messages that the actor can handle.
 pub struct MultithreadRuntimeHandle<A> {
     pub(super) sender: crossbeam_channel::Sender<MultithreadRuntimeMessage<A>>,
+    /// This is used in the case where the handle controls the lifetime of the runtime,
+    /// dropping (all of) which automatically stops the runtime, as an alterative of having
+    /// the ActorSystem control it.
+    cancellation_signal_holder: Option<crossbeam_channel::Sender<()>>,
```

Comment:
Just confirming, what's happening here is that as soon as all instances of the Sender are dropped, we would signal the `recv(cancellation_signal)` in the `crossbeam_channel::select!` to break/return?

Reply:
If so, it makes sense to add this in the comment that this is the same sender as what's used in `cancellation_signal`